src/clusterfuzz/_internal/bot/init_scripts/init_runner.py

```diff
@@ -33,6 +34,8 @@ def _extension(platform):
 def run():
   """Run custom platform specific init scripts."""
   platform = environment.platform().lower()
+  if tasks.SUBQUEUE_IDENTIFIER in platform:
+    platform = 'android'
```

Comment:
I introduced the new function `environment.base_platform()` to extract the base_platform.

However, the function is very simple (1-line). I do prefer the way that the code reads, although let me know if you prefer the just to extract the base_platform directly instead of introducing the new function.